vg_chart_generator/setup.py

```diff
@@ -0,0 +1,22 @@
+try:
+    from setuptools import setup
+except ImportError:
+    from distutils.core import setup
+
```

Comment:
So line 1-4 we are trying to import setup, but if that fails with an ImportError then it will take setup from distultils.core... Where is distutils.core? Is it included with python?